cmd/crc/cmd/start.go

```diff
@@ -129,6 +129,10 @@ type startResult struct {
 
 func (s *startResult) prettyPrintTo(writer io.Writer) error {
 	if s.Error != nil {
+		var e *crcErrors.PreflightError
+		if errors.As(s.Error, &e) {
+			logging.Warn("Preflight checks failed during `crc start`,  please try to run `crc setup` first in case you haven't done so yet")
```

Comment:
extra space before `please`. 

I did a make release and run crc start on a clean installation and the message was already clear:

```
$ crc start
Run 'crc setup' to unpack the bundle to disk
```

Maybe we can rephrase the last part and say `please run `crc setup` to configure your machine.`

Reply:
> please run crc setup to configure your machine.

@guillaumerose I like the current one because many time it happen that as user did crc setup and still preflight check fails for whatever reason so better to put `in case you haven't done so yet`.

Reply:
> extra space before please.

@guillaumerose done.